core/textile/sync/restore.go

```diff
@@ -0,0 +1,40 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/FleekHQ/space-daemon/log"
+)
+
+// replicate a local thread on the hub
```

Comment:
This is more so replicated hub data locally right?

Reply:
@jsonsivar 

Yes, fixed in https://github.com/FleekHQ/space-daemon/pull/207/commits/2bcda3d1052a5da450071ec4301631a821e8b0ff#diff-e4729a5c7a413a5b91b5b88b329f166525e3b3daec3b74e4991f7a19e1165c70R9 and, subsequently, in https://github.com/FleekHQ/space-daemon/pull/207/commits/8d0157394c577cfa40622fe5d94f4c7b11e8e90c#diff-e4729a5c7a413a5b91b5b88b329f166525e3b3daec3b74e4991f7a19e1165c70R10